### ISSUE TITLE
Static Vector Methods on cdn.js calling get() instead of copy()

### ISSUE THREAD

**Issue by rybotron** [lmccart/p5.js#534]
I found a discrepancy in the static vector methods between version 0.4.0 on github and the version on the cdn. The github version looks to be the correct version using p5.Vector.copy()) instead of p5.Vector.get(). For example:

#### CDN
```javascript
p5.Vector.div = function (v, n, target) {
    if (!target) {
      target = v.get();
    } else {
      target.set(v);
    }
    target.div(n);
    return target;
  };
```

#### Github
```javascript
p5.Vector.div = function (v, n, target) {
    if (!target) {
      target = v.copy();
    } else {
      target.set(v);
    }
    target.div(n);
    return target;
  };
````

Thanks for all the great work!

**Comment by lmccart** [comment]
thanks @rybotron, I'll package up and post a new version asap :)

**Comment by rybotron** [comment]
Thanks @lmccart!

**Comment by rybotron** [comment]
Whoops, this time being my first for reporting an issue on github I forgot to mention the actual important part is it causes a console error that get() is undefined. So the static operators aren't working on the CDN version. Thanks again!

**Comment by lmccart** [comment]
should be fixed with v0.4.1
